safenode/src/network/error.rs

```diff
@@ -36,9 +36,12 @@ pub enum Error {
     #[error("Kademlia Store error: {0}")]
     KademliaStoreError(#[from] kad::store::Error),
 
-    #[error("The mpsc::receiever has been dropped")]
-    ReceieverDropped(#[from] mpsc::SendError),
+    #[error("The mpsc::receiever for `NetworkEvent` has been dropped")]
+    NetworkEventReceieverDropped(#[from] mpsc::error::SendError<NetworkEvent>),
```

Comment:
There's a little typo here for both of these. `NetworkEventReceieverDropped` -> `NetworkEventReceiverDropped`.